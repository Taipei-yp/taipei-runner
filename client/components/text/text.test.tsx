import React from "react";
import TestRenderer from "react-test-renderer";
import { Text } from ".";

describe("Component Text", () => {
  const text = TestRenderer.create(
    <Text text="test" color="light" size="l" className="testclass" />,
  );

  it("correct render", () => {
    expect(text.toJSON()).toMatchSnapshot();
  });
  it("correct props", () => {
    expect(text.root.props.color).toBe("light");
    expect(text.root.props.size).toBe("l");
    expect(text.root.props.className).toBe("testclass");
  });
});
