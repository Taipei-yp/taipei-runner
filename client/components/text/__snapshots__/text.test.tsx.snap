// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Text render 1`] = `
<span
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
>
  test
</span>
`;
